INSS.FIP.Web/ViewModels/SearchParametersViewModel.cs

```diff
@@ -31,7 +32,7 @@ public class SearchParametersViewModel : IValidatableObject
     public string? LastName { get; set; }
 
     [StringLength(100, ErrorMessage = StringLengthValidationError)]
-    [RegularExpression(RegExForName, ErrorMessage = InvalidCharactersValidationError)]
+    [RegularExpression(RegExForCompanyName, ErrorMessage = InvalidCharactersValidationError)]
```

Comment:
Could you add unit test for the fix

Reply:
Added some unit test cases for this and improved the existing one as well.